mlrun/kfpops.py

```diff
@@ -38,7 +39,7 @@
     run_keys,
 )
 
-KFPMETA_DIR = os.environ.get("KFPMETA_OUT_DIR", "")
+KFPMETA_DIR = os.environ.get("KFPMETA_OUT_DIR", "/tmp")
 KFP_ARTIFACTS_DIR = os.environ.get("KFP_ARTIFACTS_DIR", "/tmp")
```

Comment:
Please add a comment regarding those two params, as well as why you changed `KFPMETA_DIR` default to "/tmp"